tests/foreman/virtwho/utils.py

```diff
@@ -109,7 +109,7 @@ def register_system(system, activation_key=None, org='Default_Organization'):
     if activation_key is not None:
         cmd += '--activationkey={}'.format(activation_key)
     else:
-        cmd += '--username={} --password={}'.format(
+        cmd += '--username={} --password={} --environment=Library'.format(
```

Comment:
Set Library as the `default` value would be better
`def register_system(system, activation_key=None, org='Default_Organization', env='Library'):`

```suggestion
        cmd += '--username={} --password={} --environment={}'.format(
```